game_view.cpp

```diff
@@ -532,7 +516,25 @@ void test_game_view()//!OCLINT tests may be many
     }
 #endif // FIX_ISSUE_246
 
-  #endif
+
+  // (545) A game window doesn't open at construction
```

Comment:
I like this style of adding the Issue number in the comment aboce the test. You'll see me copy it :-)